collector/README.md

```diff
@@ -156,6 +156,38 @@ If you've collected new data, you can run `curl -X POST
 localhost:2346/perf/onpush` to update the site's view of the data, or just
 restart the server.
 
+### Benchmarking on Windows
+
+To benchmark on Windows, you will need to run the collector in a elevated context
+so that it can access the hardware performance counters. Note: some virtualized
+environments do not permit access to these counters for guest VMs.
+
+You will also need to provide the paths to the xperf and tracelog tools (or have them
+available on your PATH). Some common paths to these tools look like:
+
+```
+$env:XPERF="C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\xperf.exe"
+$env:TRACELOG="C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x64\tracelog.exe"
+```
+
+Finally, while most of the options you can pass to the collector are supported, the majority of
+the profilers are not. In Windows, the only currently supported profiler is the `self-profiler`.
```

Comment:
Based on the code/implementation it seems like xperf-stat might be supported too?

Reply:
I was more referring to the list of profilers supported by the `perf_local` command which doesn't support `perf-stat` either (but does support a `perf` profiler). I think that probably makes sense to support an `xperf` profiler there but that's not currently implemented.

Reply:
Ah, makes sense. Could you clarify this here? I think it's generally a confusing piece of our infrastructure today but seems good to not add to that!

Reply:
Sure! Let me know if you think I should add more.